ResearchKit/Consent/ORKVisualConsentStepViewController.m

```diff
@@ -661,13 +662,12 @@ - (NSUInteger)indexOfViewController:(UIViewController *)viewController {
         return NSNotFound;
     }
     
-    NSUInteger index = NSNotFound;
     for (NSNumber *key in _viewControllers) {
         if (_viewControllers[key] == viewController) {
-            index = [key unsignedIntegerValue];
+            return [key unsignedIntegerValue];
```

Comment:
We have three logical returns within the scope of the method.  To put the break back in on this one, we would need to restore the NSUInteger to serves as a primitive container, vs the latter (hair lighter on memory and faster) saves us the container, setter, break, and return vs return.   (May not be needed within this scope, however adopted pattern long time ago from mathematically modeling and realized performance gains.)  Intention was just to save the "NSUInteger index = NSNotFound;, that may not be needed.